pkg/operator/starter.go

```diff
@@ -176,6 +177,7 @@ var deploymentConfigMaps = []revision.RevisionResource{
 	{Name: "config"},
 	{Name: "controller-manager-kubeconfig"},
 	{Name: "serviceaccount-ca"},
+	{Name: "service-ca", Optional: true},
```

Comment:
if you're checking length and statically sync, then you don't need to be optional.

Reply:
bc12f357